resources/styles/question.less

```diff
@@ -0,0 +1,69 @@
+.question {
+	.move-question {
+		float: right;
+		font-size: 2rem;
+		margin: 0 10px;
+	}
+
+	.remove-question {
+		float: right;
+		color: @red;
+		margin-top: 0.7rem;
+		i {
+			margin-right: 10px;
+		}
+	}
+  textarea {
+    display: block;
```

Comment:
Nope, actually `<textarea>` defaults to `inline-block`.  With that said though, I don't see why it needs to be set to `block`, it still renders fine when I remove the rule.  This file is extracted from when all styles where just one big file, so I'm not sure of the reasoning for setting it to `block`